### PR TITLE
Twilio decommission

### PR DESCRIPTION
**Pull request by acodegirl**
Deactivated Twilio from providers list.

### CODE REVIEW

File: api/app/dao/provider_details_dao.py [hunk 1]
```diff
@@ -27,20 +27,19 @@ def get_provider_details_by_identifier(identifier):
...
-    if os.getenv('FEATURE_SAP_ENABLED'):
-        if identifier == 'sap':
-            alternate_provider = 'telstra'
-        elif identifier == 'telstra':
-            alternate_provider = 'sap'
-        # We need to have a third check here because our tests expect that
-        # Twilio is enabled.
-        elif identifier == 'twilio':
-            alternate_provider = 'sap'
+    if identifier == 'telstra':
```
Comment (dagoof) [inline api/app/dao/provider_details_dao.py:30]: can we change the logic in here a bit? something like:

```
if identifier != 'sap':
  alternate_provider = 'sap'
else:
  alternate_provider = 'telstra'
```

Comment (acodegirl) [reply api/app/dao/provider_details_dao.py:30]: I was not sure whether to remove Twilio case here. I will change the logic and update.

File: api/app/dao/provider_details_dao.py [hunk 2]
```diff
@@ -27,20 +26,10 @@ def get_provider_details_by_identifier(identifier):
 
 def get_alternative_sms_provider(identifier):
     alternate_provider = None
-    if os.getenv('FEATURE_SAP_ENABLED'):
```
Comment (dagoof) [inline api/app/dao/provider_details_dao.py:30]: one last thing -- can we remove references to `FEATURE_SAP_ENABLED` from `api/manifest.yml` and `test_provider_details_dao`

Comment (acodegirl) [reply api/app/dao/provider_details_dao.py:30]: done

**Pull request merged**